app/views/cloud_object_store_container/new.html.haml

```diff
@@ -0,0 +1,62 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "cloudObjectStoreContainerFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Basic Information')
+  .form-horizontal
+    .form-group{"ng-class" => "{'has-error': angularForm.storage_manager_id.$invalid}"}
+      %label.col-md-2.control-label
+        = _('Storage Manager')
+      .col-md-8
+        = select_tag("storage_manager_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @storage_manager_choices.sort, disabled: ["<#{_('Choose')}>", nil]),
```

Comment:
In an effort to make our angular forms that add  new functionality (like this one) make use of REST API, could we populate this dropdown using a REST API call as well instead of using the instance variable `@storage_manager_choices`?

Can you try out this API and let me know if it works well for this dropdown?

`/api/providers?expand=resources&filter[]=type=ManageIQ::Providers::Amazon::StorageManager::S3`